Silicon/CoffeelakePkg/Library/PlatformHookLib/PlatformHookLib.c

```diff
@@ -66,7 +66,7 @@ GetSerialPortBase (
 
   DebugPort = GetDebugPort ();
   if (DebugPort >=  PCH_MAX_SERIALIO_UART_CONTROLLERS) {
-    if (DebugPort == 0xFE) {
+    if (DebugPort == 0x80) {
```

Comment:
why do we change this ?

Reply:
using DebugPort (a byte) to know which serial device  (ISA serial or PCH serial) and port number. If BIT7 is set, it is ISA serial device, else it is PCH serial device. other bits are used for port number, for ISA serial, 0 means 0x2F8 device, 1 means 0x3F8 device.

Reply:
0xFE,0xFF has been documented on the github wiki pages and the stitching scripts for CFL.   Unless it is absolutely required,  I would like to leave it and let the platform to decide the meaning for the debug port number.